stack.yaml

```diff
@@ -1,5 +1,6 @@
-resolver: lts-13.26
+resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/3ac22a2fda11ca7131a011a9ea48fcbfdc26d6b3/snapshot.yaml
```

Comment:
we should not depend on cardano-prelude

Reply:
Done, let's hope it builds with `lts-13.26`.